### PR TITLE
Forgot shroud on all-cheat

### PR DESCRIPTION
**Pull request by pevers**
I forgot the disableshroud on the "all" cheat. I also merged some lines together in one assignment.

### CODE REVIEW

File: OpenRA.Game/Traits/Player/DeveloperMode.cs [hunk 1]
```diff
@@ -70,32 +70,22 @@ public void ResolveOrder(Actor self, Order order)
...
-
-							EnableAll = true;
 						}
 						else
 						{
-							AllTech = false;
-							FastCharge = false;
-							FastBuild = false;
-							DisableShroud = false;
+							EnableAll = AllTech = FastBuild = DisableShroud = UnlimitedPower = BuildAnywhere = false;
```
Comment (jabbink) [inline OpenRA.Game/Traits/Player/DeveloperMode.cs]: You forgot to reset FastCharge to false here

**Comment by jabbink** [comment]
Squashed the commits

File: OpenRA.Game/Traits/Player/DeveloperMode.cs [hunk 2]
```diff
@@ -68,34 +68,24 @@ public void ResolveOrder(Actor self, Order order)
 			{
 				case "DevAll":
 					{
-						if (!EnableAll)
+						EnableAll ^= true;
+						AllTech = FastCharge = FastBuild = DisableShroud = UnlimitedPower = BuildAnywhere = EnableAll;
+						if (EnableAll)
```
Comment (Phrohdoh) [inline OpenRA.Game/Traits/Player/DeveloperMode.cs]: Please add an empty line above this one.

File: OpenRA.Game/Traits/Player/DeveloperMode.cs [hunk 3]
```diff
@@ -68,34 +68,25 @@ public void ResolveOrder(Actor self, Order order)
 			{
 				case "DevAll":
 					{
-						if (!EnableAll)
+						EnableAll ^= true;
+						AllTech = FastCharge = FastBuild = DisableShroud = UnlimitedPower = BuildAnywhere = EnableAll;
+
```
Comment (obrakmann) [inline OpenRA.Game/Traits/Player/DeveloperMode.cs]: travis says:
> OpenRA.Game/Traits/Player/DeveloperMode.cs:L73: [SP2000] Invalid spacing at the end of the line.

**Comment by Mailaender** [comment]
Works as promised. :white_check_mark:

**Comment by obrakmann** [comment]
:+1:

**Comment by obrakmann** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/_compare/2f2ee83694e838964d86f8744d46cda8c2d2f3d2%5E...2f2ee83694e838964d86f8744d46cda8c2d2f3d2)

**Pull request merged**